.yarnrc

```diff
@@ -1,2 +1,2 @@
 # Use Yarn 2
-yarn-path ".yarn/releases/yarn-sources.js"
+yarn-path ".yarn/releases/yarn-rc.js"
```

Comment:
The `yarn-rc.js` file is committed, so that:
1. We all use the same yarn version
2. You can have whatever yarn version you want installed on your PC.